#Message where each sub command from simulink or trigger script for the MQS is published into

uint8[16] auto_ctrls
